app.json

```diff
@@ -0,0 +1,32 @@
+{
+  "buildpacks": [
+    {
+      "url": "heroku-community/apt"
+    },
+    {
+      "url": "heroku/nodejs"
```

Comment:
Do we need this buildpack? If so, what for?

Reply:
We do, assets:precompile would fail if the `yarn` binary isn't installed, this buildpack does the trick.